substitutions.sh

```diff
@@ -0,0 +1,11 @@
+mkdir subs && cd subs && touch File{A..F}.txt
+rm File{C..F}.txt
+mv FileA.sh FileA.old
+rm *
+cp /etc/*.conf subs 2>/dev/null
+cat l*
+grep -l "user" ????.[^c][^o][^n][^f]
```

Comment:
Not sure why use these marking in the file extension. 
Just `grep -l "user" ????.conf` is good enough